dags/bqetl_unified.py

```diff
@@ -47,6 +47,23 @@
     tags=tags,
 ) as dag:
 
+    telemetry_derived__rolling_cohorts__v1 = bigquery_etl_query(
```

Comment:
Do we want to group cohorts in bqetl_analytics_aggregations, or you prefer here because is not an aggregation?

Reply:
Yup exactly, will put `cohort_daily_statistics` in `bqetl_analytics_aggregations`